### ISSUE TITLE
Mints and burns entity?

### ISSUE THREAD

**Issue by greecychipmunk** [curvefi/volume-subgraphs#5]
Hello, Would it be possible to include an entity showing mint and burn events?

In an old (staging) version of the subgraph, something like this used to exist, though it didn't seem complete.  It was called LiquidityEvent, I think.

Ideally, it would show the addresses of the tokens that were added or removed, the address of the pool, the timestamp (of course), what amounts were added / removed, and convert each amount to USD as an aid to the user who would probably need to make this conversion.
(The old version had the total USD amount of all tokens combined, but it wasn't clear how to compute each token's value individually.)

Thank you very much.  Sorry, i don't have the expertise to help, I'd just like to use it if possible.  
Thank you for what you have developed already!

**Comment by benber86** [comment]
Indeed we used to have this, on an [older version of the subgraph](https://thegraph.com/hosted-service/subgraph/convex-community/curve-factory-volume). This was originally added by @bout3fiddy and removed when we made the subgraph crosschain due to concerns that it would slow down the syncing too much. 

However I don't think the feature would actually be such a burden after all, so tagging fiddy in case he has the time to add it back and test it on all chains at some point.
